engine/tester/config_parse.c

```diff
@@ -2713,8 +2733,11 @@ get_test_package(xmlNodePtr root, tester_cfg *cfg,
 
     node = root->children;
 
-    /* Get mandatory description */
-    if (node != NULL &&
+    if (pkg->objective != NULL)
```

Comment:
Also it would be nice to add a comment which explains what is happening here.
"Get mandatory description" is insufficient.